pallet-chainlink-feed/src/lib.rs

```diff
@@ -964,7 +963,7 @@ pub mod pallet {
 				.ok_or(Error::<T>::InsufficientFunds)?;
 
 			T::Currency::transfer(
-				&Self::into_account(feed_id),
+				&T::PalletId::get().into_account(),
```

Comment:
We could add a designated `account_id()` function that returns this, since we're using this a couple of times:

```rust
fn account_id() -> T::AccountId {
		T::PalletId::get().into_account()
}
```